heat/core/linalg/tests/test_lanczos.py

```diff
@@ -0,0 +1,87 @@
+import torch
+import os
+import unittest
+import heat as ht
+import numpy as np
+
+if os.environ.get("DEVICE") == "gpu" and torch.cuda.is_available():
```

Comment:
See above, necessary in every test class?

Reply:
only if we want to be able to run a specific test class separate from the rest, @mtar has implemented a function in pr #488 for the device selection that we can use in the future